package.json

```diff
@@ -38,9 +37,10 @@
     "tsd": "^0.11.0"
   },
   "dependencies": {
+    "@types/ws": "^7.2.2",
```

Comment:
I would highly prefer to avoid adding this external dependency for JS devs

Reply:
Me too. But the type from `ws` will be `any` for TS devs.

Reply:
They can always install that package if they need it.